staging/src/kubevirt.io/client-go/api/v1/schema.go

```diff
@@ -417,6 +420,19 @@ type Filesystem struct {
 	Virtiofs *FilesystemVirtiofs `json:"virtiofs"`
 }
 
+//
+// +k8s:openapi-gen=true
+type Sound struct {
+	// Mandatory attribute, which specifies what real sound device is emulated.
+	// Supported values:  es1370, sb16, ac97, ich6, usb.
+	// More info: https://libvirt.org/formatdomain.html#sound-devices
+	Model string `json:"model"`
+	// Audio codecs attached to the audio device.
+	// Supported values: duplex, micro, output.
+	// +optional
+	CodecType string `json:"type,omitempty"`
```

Comment:
The comments above each struct need to start with the struct's name.

So for Model the comment would need to be something like

```
// Model specifies what real sound device is emulated
// Supported values: es1370, sb16, ac97, ich6, usb.
```